classytags/utils.py

```diff
@@ -83,8 +83,9 @@ def mixin(parent, child, attrs={}):
 def flatten_context(context):
     if callable(getattr(context, 'flatten', None)):
         return context.flatten()
-    else:
+    elif hasattr(context, 'dicts'):
```

Comment:
I'm not convinced that this is the proper way to do this, though I haven't fully looked into it. Maybe it should instead be ``elif isinstance(context, Context)`` (or ``isinstance(context, dict)``?). I also noticed there's no test for this function directly, which is my fault and I'll try to fix this possibly tonight when I get off work. But I think I can take it from here, so you don't have to worry about this too much.